### PR TITLE
6443 Audio Menu Sample Sound won't play while domain is loading

### PR DESCRIPTION
**Pull request by luiscuenca**
This PR preloads the sample audio when the application starts so that it's always available by the time the menu loads.

https://highfidelity.fogbugz.com/f/cases/6443

...

TEST PLAN

- Go to Welcome or any other domain with a lot of content
- While it is loading, open the audio menu and play the sample audio
- Activate the advanced and developer menus and go to edit ->clear cache
- Repeating this process should have the same outcome. (sample audio always plays)

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/4429/HighFidelity-Beta-PR11681-a1c9285c902be06cd7220f9bcf9b0c77601099c7.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/4429/HighFidelity-Beta-PR11681-a1c9285c902be06cd7220f9bcf9b0c77601099c7.exe

File: interface/resources/qml/hifi/audio/PlaySampleSound.qml [hunk 1]
```diff
@@ -22,16 +22,15 @@ RowLayout {
...
+        sound = SampleSound;
         sample = null;
     }
     function playSound() {
         // FIXME: MyAvatar is not properly exposed to QML; MyAvatar.qmlPosition is a stopgap
         // FIXME: Audio.playSystemSound should not require position
         if (sample === null && !isPlaying) {
             sample = Audio.playSystemSound(sound, MyAvatar.qmlPosition);
-            isPlaying = true;
+			isPlaying = true;
```
Comment (sethalves) [inline interface/resources/qml/hifi/audio/PlaySampleSound.qml]: indentation change here

File: interface/src/Application.cpp [hunk 2]
```diff
@@ -1787,9 +1793,9 @@ Application::Application(int& argc, char** argv, QElapsedTimer& startupTimer, bo
         return entityServerNode && !isPhysicsEnabled();
     });
 
-    QFileInfo inf = QFileInfo(PathUtils::resourcesPath() + "sounds/snap.wav");
-    _snapshotSound = DependencyManager::get<SoundCache>()->getSound(QUrl::fromLocalFile(inf.absoluteFilePath()));
-
+    QFileInfo inf_snap = QFileInfo(PathUtils::resourcesPath() + "sounds/snap.wav");
```
Comment (sethalves) [inline interface/src/Application.cpp]: coding standard insists on camelCase rather than snake_case

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/4457/HighFidelity-Beta-PR11681-b7c375f396371c1082b16f5439a7dc50fd3ca2a5.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/4457/HighFidelity-Beta-PR11681-b7c375f396371c1082b16f5439a7dc50fd3ca2a5.exe

**Comment by cain-kilgore** [comment]
Testing.

**Comment by cain-kilgore** [comment]
# Test Passed
Test | Result
--- | ---
Go to Welcome or any other domain with a lot of content | Pass
While it is loading, open the audio menu and play the sample audio | Pass
Activate the advanced and developer menus and go to edit ->clear cache | Pass
Repeating this process should have the same outcome. (sample audio always plays) | Pass

Perfect!

###### Intel i5 7600K @ 4Ghz, 16GB DDR4 2133Mhz Corsair Vengeance, Asus Strix GTX 1050Ti 3GB, HTC Vive

**Pull request merged**